### PR TITLE
Fixes #3193 - note datetime conversion

### LINKED ISSUE #3193: Document that the -as operator is (unexpectedly) culture-sensitive, in about_Type_Operators
_Casts_, which - by design - always use the _invariant culture_, so that you can use something like the following, irrespective of the current culture:

```powershell
[datetime] '12/30/2018'  # OK, irrespective of current culture
```

By contrast, when using `-as` with a string as the LHS, _culture-sensitive_ parsing is invariably applied:

```powershell
'12/30/2018' -as [datetime] # OK *only* in culture en-US or similar

# NO output ($null), because culture fr-FR (French) does not recognize this date format.
PS> [cultureinfo]::CurrentCulture = 'fr-FR'; '12/30/2018' -as [datetime]
```

Unfortunately, the behavior of `-as` is _internally inconsistent_, because other types, such as `[double]`, again is culture-_invariant_ (I don't know what rules underlie this discrepancy):

```powershell
PS> [cultureinfo]::CurrentCulture = 'fr-FR'; '1,2.3' -as [double]
12,3  # '1,2.3' was recognized, implying that the INVARIANT culture was used.
```

This surprising discrepancy is likely unintentional, but fixing it may be too much of  a breaking change - see https://github.com/PowerShell/PowerShell/issues/8129

Either way, it should be documented for the existing versions.

Version(s) of document impacted
------------------------------
- [x] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Fixes #3193 - note datetime conversion
Fixes [AB#1699468](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1699468)

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.x preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://docs.microsoft.com/powershell/scripting/community/contributing/overview) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Type_Operators.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-The Boolean type operators (-is and -isNot) tell whether an object is an
-instance of a specified .NET Framework type. The -is operator returns a
-value of TRUE if the type matches and a value of FALSE otherwise. The
--isNot operator returns a value of FALSE if the type matches and a value of
-TRUE otherwise.
+The Boolean type operators (`-is` and `-isNot`) tell whether an object is an
+instance of a specified .NET type. The `-is` operator returns a value of TRUE
+if the type matches and a value of FALSE otherwise. The `-isNot` operator
+returns a value of **FALSE** if the type matches and a value of **TRUE**
+otherwise.
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Type_Operators.md]: Both sets of TRUE FALSE should be bolded the same.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Script_Internationalization.md [hunk 2]
```diff
@@ -33,18 +33,18 @@ features:
   information about the Data section, see
   [about_Data_Sections](about_Data_Sections.md).
 
-* New automatic variables, \$PSCulture and \$PSUICulture. \$PSCulture stores the
-  name of the UI language used on the system for elements such as the date,
-  time, and currency. The $PSUICulture variable stores the name of the UI
-  language used on the system for user interface elements such as menus and text
-  strings.
+* New automatic variables, `$PSCulture` and `$PSUICulture`. `$PSCulture` stores
```
Comment (DCtheGeek) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Script_Internationalization.md]: Some missed `*` to replace with `-` for bullets.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Type_Operators.md (hunk 1 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Type_Operators.md]: fixed

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Script_Internationalization.md (hunk 2 above)
Comment (sdwheeler) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Script_Internationalization.md]: Fixed

**Pull request merged**